test/mappers/http-error-mapper.test.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+const { HttpError } = require('@uphold/http-errors');
+const HttpErrorMapper = require('src/mappers/http-error-mapper');
+const GrpcError = require('grpc-error');
+
+/**
+ * Test `HttpErrorMapper`.
+ */
+
+describe('HttpErrorMapper', () => {
+  const mapper = new HttpErrorMapper();
+
+  describe('map()', () => {
+    it('should return `undefined` if the error is not supported', () => {
+      const result = mapper.map(new Error());
+
+      expect(result).toBe(undefined);
+    });
+
+    it('should map to `UNKNOWN` error if the code is not handled', () => {
+      const error = new HttpError(-2);
+
+      try {
+        mapper.map(error);
+
+        fail();
+      } catch (e) {
+        expect(e).toBeInstanceOf(GrpcError);
+        expect(e.code).toEqual(2);
```

Comment:
Since we still have `grpc` in `devDependencies`, we can use it in tests. So WDYT about using the `status.ENUM` values in tests? Would make for an opportunity to verify the hardcoded numeric values match our expectations.